### ISSUE TITLE
Crash Constructed last turn frostbolt to my own face while enemy conceded

### ISSUE THREAD

**Issue by mickvangelderen** [Epix37/Hearthstone-Deck-Tracker#1571]
```
########## 2015-10-07 17:53:44 ##########
System.NullReferenceException: Object reference not set to an instance of an object.
   at Hearthstone_Deck_Tracker.GameEventHandler.<HandleGameEnd>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<ThrowAsync>b__4(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by Epix37** [comment]
same as #1531 - probably fixed with v0.11.4 pre-release.

**Comment by mickvangelderen** [comment]
My bad, I read through that issue switfly and throught it only concerned tavern brawls.
